distributed/scheduler.py

```diff
@@ -5597,6 +5613,9 @@ def heartbeat_interval(n):
         return n / 200 + 1
 
 
+
+
+
```

Comment:
You'll need to install the git pre-commit hooks.  See https://docs.dask.org/en/latest/develop.html#code-formatting